.github/workflows/packaging-test-pipelines.yml

```diff
@@ -0,0 +1,166 @@
+name: Build tests in packaging images
+
+on:
+  push:
+    branches: "**"
+
+  workflow_dispatch:
+
+jobs:
+
+  get_postgres_versions_from_file:
+    runs-on: ubuntu-latest
+    outputs:
+      pg_versions: ${{ steps.get-postgres-versions.outputs.pg_versions }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 2
+      - name: Get Postgres Versions
+        id: get-postgres-versions
+        run: |
+          ls -la .github
+          # Postgres versions are stored in .circleci/config.yml file in "build-[pg-version] format. Below command
+          # extracts the versions and get the unique values.
+          pg_versions=`grep -Eo 'build-[[:digit:]]{2}' .circleci/config.yml|sed -e "s/^build-//"|sort|uniq|tr '\n' ','| head -c -1`
+          pg_versions_array="[ ${pg_versions} ]"
+          echo "Supported PG Versions: ${pg_versions_array}"
+          # Below line is needed to set the output variable to be used in the next job
+          echo "pg_versions=${pg_versions_array}" >> $GITHUB_OUTPUT
+
+  build_tests_for_rpm_in_packaging_images:
+    name: tests_build_in_packaging_images
```

Comment:
Can you have shorter names here? It gets hard to understand what failed when the name gets truncated.

![image](https://user-images.githubusercontent.com/6772247/204993043-0d4193fe-1547-4a19-8134-6354c6609e1e.png)

I think the whole name reads: `tests_build_in_packaging_images (citus/packaging:centos-8-pg, 15)` The `centos-8-pg-15` is the important part that should be visible

Reply:
@gurkanindibay You should have updated `name:` field. The interface is still the same.

Reply:
Yes you're faster than me. Now updated :)

Reply:
![image](https://user-images.githubusercontent.com/6772247/205034332-856bb335-e9b0-4693-b4d5-b749055bd69d.png)

It is getting better, but still it does not have a nice UX. Can you get rid of the `citusdata/packaging:` on each line by moving it out from the matrix and possibly parameterizing it?

Reply:
See suggestion in https://github.com/citusdata/citus/pull/6499#discussion_r1037013990

Reply:
Fixed now